app/frontend/actions.ts

```diff
@@ -375,14 +169,14 @@ export default (store: Store) => {
     const newState = getState()
     try {
       if (newState.sendTransactionSummary.plan) {
-        txAux = await wallet
+        txAux = await getWallet()
           .getAccount(state.sourceAccountIndex)
```

Comment:
:scream: why are we mixing state and newState at all? :confused:

Reply:
Maybe let's discuss this on slack call? I am not sure whether your comment is about the refactor / current state or something else.